### ISSUE TITLE
Exclude `Strauss Road Bridge` assets

### ISSUE THREAD

**Issue by originalfoo** [algernon-A/CurbHeightAdjuster#7]
Assets:

* https://steamcommunity.com/sharedfiles/filedetails/?id=2566350479
* https://steamcommunity.com/sharedfiles/filedetails/?id=2553884716

While the curb height works great on the actual roads, it doesn't work on the moveable bridge (a building with CAL):

![image](https://user-images.githubusercontent.com/1386719/152270665-25bc9381-f51d-4e99-8665-cfb83d0f9e14.png)

I'm assuming the bridge is just a static mesh and thus not possible to "curb height' it...

![image](https://user-images.githubusercontent.com/1386719/152270688-afdb3272-cbb2-4c0e-bf3d-653464d4c44a.png)

**Comment by algernon-A** [comment]
Yes, on first glance that's a special mesh - but can investigate further.  It might (probably will) require custom treatment, but if it's a mesh then I *should* be able to alter it.

I'll come back to this once I've finished the overpass-and-bridge-deck-depth-thickness-shrinking functionality.
